libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,21 +7,21 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button type="search" aria-label = "Search Books" mat-icon-button matSuffix>
       <mat-icon>search</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
 <ng-container *ngIf="searchTerm; else empty">
   <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
-      <div class="book--title">
+    <div class="book" data-testing="book-item" *ngFor="let b of this.books$|async">
```

Comment:
```suggestion
    <div class="book" data-testing="book-item" *ngFor="let b of books$|async">
```
Referencing `this` in the template will work but in Angular, we just leave it off in our template code by convention.